src/editor/ZoomControls.tsx

```diff
@@ -42,29 +42,24 @@ const ZoomControls = ({ size, ...props }: ZoomControlsProps) => {
   }, [setSettings, settings]);
   const intl = useIntl();
   return (
-    <ButtonGroup
-      {...props}
-      isAttached
-      colorScheme="gray"
-      variant="zoom"
-      transform="rotate(90deg)"
-      transformOrigin="bottom"
-    >
+    <ButtonGroup {...props} colorScheme="blackAlpha" variant="ghost">
       <IconButton
         size={size}
         isRound
-        icon={<RiZoomInLine style={{ transform: "rotate(-90deg)" }} />}
-        aria-label={intl.formatMessage({ id: "zoom-in-action" })}
-        onClick={handleZoomIn}
+        color="#838383"
```

Comment:
It'd be nice to sort this out before merging.

Reply:
OK, I agree this doesn't fit nicely into the current greys. The scale is a bit of a mess and I'd like to revisit based on the new greys, but as it'll affect more than this PR I'll do it separately.